### ISSUE TITLE
glyph overlap in Segoe UI Emoji

### ISSUE THREAD

**Issue by emojifreak** [khaledhosny/harftex#12]
```
\documentclass{article}
\usepackage{harfload}
\usepackage{fontspec}

\begin{document}
\Huge
\noindent
\fontspec{Segoe UI Emoji}[RawFeature={mode=harf}]
👨‍👩‍👧‍👦⏰⏰\\
\fontspec{Noto Color Emoji}[RawFeature={mode=harf}]
👨‍👩‍👧‍👦⏰⏰
\end{document}
```

When I process the above file with harflatex 0.40 installed from TeXLive Contrib, I get
![emoji4](https://user-images.githubusercontent.com/53028623/63004298-02645a80-beb5-11e9-87aa-47150f150cea.png)
We see that the glyphs in Segoe UI Emoji are overlapping. I doubt if this is an intended behavior. The generated PDF is attached below.
[emoji4.pdf](https://github.com/khaledhosny/harftex/files/3500351/emoji4.pdf)

**Comment by khaledhosny** [comment]
Is this a regression, did you get proper spacing before?

**Comment by emojifreak** [comment]
@khaledhosny Thank you for your reply.
No, I believe it is NOT a regression. I have seen it before but did not see it as something to be reported. Moreover, `mode=node` and `lualatex` also give almost the same (improper) spacing with the black-and-white version of Segoe UI Emoji. Below is the result with `mode=node`. I am sorry if this is a bug (or a feature) of the font.
![Screenshot at 2019-08-15 04-17-37](https://user-images.githubusercontent.com/53028623/63049463-b5fc3780-bf13-11e9-9170-1a2d17888578.png)

**Comment by khaledhosny** [comment]
Looks like a HarfBuzz issue. I can reproduce the bad spacing with other HarfBuzz applications, but not with Apple’s TextEdit. Can you check on Windows whether applications like Notepad, Wordpad and MS Word handle this correctly?

The font has a `dist` feature and seems to expect it to be tutned on by default, but HarfBuzz only enables it by default for certain scripts only. Enabling it manually fixes the spacing:
```tex
\fontspec{Segoe UI Emoji}[RawFeature={mode=harf,dist}]
```

**Comment by emojifreak** [comment]
@khaledhosny Windows Notepad and Microsoft Word make correct spacing, i.e., two spacing among three glyph are the same in Segoe UI Emoji. Firefox on Windows make incorrect spacing, as expected.

**Comment by khaledhosny** [comment]
Thanks, this will be fixed in HarfBuzz.

**Comment by khaledhosny** [comment]
Fixed with 5645b9f74dae9476f80147b2174ae2e6966bddb6.
